cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -326,6 +328,10 @@ func (c *Client) RemoveMember(id uint64) ([]Member, error) {
 		if err == nil {
 			return true, nil
 		}
+		if errors.Is(rpctypes.ErrMemberNotFound, err) {
+			klog.V(5).Infof("Member was already removed, because member %016x was not found", id)
```

Comment:
```suggestion
			klog.V(5).Infof("Member was already removed, because member %s was not found", strconv.FormatUint(id, 16))
```
Member id does not necessarily seems to be 16-bit.  We can use `strconv.FormatUint(id, 16)` here to format it, just like what `etcd` is doing:
https://github.com/kubernetes/kubernetes/blob/8f3f997a98c7b44925f7654e5c66e6f46e328147/vendor/go.etcd.io/etcd/client/pkg/v3/types/id.go#L24C7-L26

Reply:
I see what you mean. `strconv.FormatUint(id, 16)` does not pad the output with zeros, whereas `%016x` does, if there are fewer than 16 bytes. (If anyone is curious, see https://go.dev/play/p/pwOK24gL4nH)

In this file, there are 5 other log messages that use `%016x`, and I want to be consistent with them.

While I agree that we want use the upstream convention (no zero padding), I think it's a change that should be made to all log messages, and so is better as a separate PR.

Reply:
Also, thanks for the review @SataQiu!